tools/pd-ctl/pdctl/command/store_command.go

```diff
@@ -201,6 +202,37 @@ func NewStoreLimitSceneCommand() *cobra.Command {
 	}
 }
 
+// NewUnsafeRecoverCommand returns the unsafe recover command for the store command.
+func NewUnsafeRecoverCommand() *cobra.Command {
+	sc := &cobra.Command{
+		Use:   "unsafe-recover [<type>]",
+		Short: "Recover failed stores unsafely",
+		Long:  "Recover failed stores unsafely, <type> can be a list of stores e.g. '<s1,s2,...>' or 'show' or 'history'",
```

Comment:
```suggestion
		Use:   "unsafe-recover <store_ids>",
		Short: "Recover failed stores unsafely",
		Long:  "Recover failed stores unsafely, <store_ids> should be separated by comma",
```
No need to say 'show' and 'history', their description would be generated automatically by `AddCommand`